tests/e2e/utils/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 - `createCoupon( couponAmount )` component which accepts a coupon amount string (it defaults to 5) and creates a basic coupon. Returns the generated coupon code.
 - `evalAndClick( selector )` use Puppeteer page.$eval to select and click and element.
 - `selectOptionInSelect2( selector, value )` util helper method that search and select in any select2 type field
+- `selectOrderAction( action )` util helper method to select and initiate an order action in the Order Action postbox
+- `merchant.openEmailLog()` go to the WP Mail Log page
```

Comment:
I'm wondering whether we should include this in the merchant flows since the email plugin won't be included in the core packages. 

@zhongruige @tammullen What do you think of making an `email` flow? If we add functions to parse order IDs, etc., they could be added to that flow.

Reply:
I debated a bit on this. I wasn't sure how tightly coupled we want the merchant flow to be to a plugin like this, so I'd be interested in feedback here. I can see the value in adding an `email` flow, especially, since this can be leveraged for the shopper flows too.

Reply:
I think this is okay for now. If we decide to make a separate flow, this one could just call the separate one.

Reply:
Sounds good!